dids/src/main/kotlin/web5/sdk/dids/DidResolvers.kt

```diff
@@ -0,0 +1,51 @@
+package web5.sdk.dids
+
+import foundation.identity.did.DID
```

Comment:
aye, it's been used [right here](https://github.com/TBD54566975/web5-kt/blob/b263cbe6c4924b6cc954d8b2b8d3eec63d7ea5e9/dids/src/main/kotlin/web5/sdk/dids/DidResolvers.kt#L34) to parse DIDs

Reply:
I meant that, it seems to me, that we're making an effort to actively reduce dependencies. For example, `web5.sdk.dids.DidResolutionResult` was created, instead of using the class defined in `uniresolver.result.ResolveDataModelResult`.

So should we treat `foundation.identity.did.DID` the same? If not, mind explaining what the differences are between this (i.e. using `foundation.identity.did.DID`) and the example above?

Reply:
well primarily i didn't see `uniresolver.result.ResolveDataModelResult` which is why i rolled my own. [I see it](https://github.com/decentralized-identity/universal-resolver/blob/main/uni-resolver-core/src/main/java/uniresolver/result/ResolveDataModelResult.java) now so thanks for pointing that out

only reason i can see for keeping the one that's already there is, after a brief glance it looks like resolution metadata and did document data aren't strongly typed. Also generally afraid of danubetech libs after running into [this](https://github.com/decentralized-identity/did-common-java/issues/23).

Reply:
Fair enough!

This is non-blocker at all btw. Just seems that now that we've invested into our own things, it would make sense to control all the code we have.